deploy-as-code/helm/charts/ifix-services/aggregator/values.yaml

```diff
@@ -1,38 +1,27 @@
 # Common Labels
 labels:
-  app: "aggregator"
+  app: "fiscal-event-aggregator"
   group: "ifix"
 
 cron:
   schedule: "0 0 * * *"  
 # Container Configs
 image:
-  repository: "aggregator"
+  repository: "fiscal-event-aggregator"
 
```

Comment:
Write default values for 
aggregator-topic-name
aggregator-datasource-name

Reply:
Added default values.